app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -14,19 +14,27 @@ class Api::V2::RepositorySetsController < Api::V2::ApiController
     api :GET, "/products/:product_id/repository_sets", N_("List repository sets for a product.")
     param :product_id, :number, :required => true, :desc => N_("ID of a product to list repository sets from")
     param :name, String, :required => false, :desc => N_("Repository set name to search on")
+    param :enabled, :bool, :required => false, :desc => N_("If true, only return repository sets that have been enabled. Defaults to false")
     param_group :search, Api::V2::ApiController
     def index
-      collection = {}
+      respond(:collection => scoped_search(index_relation, "id", :asc, :custom_sort => :default_sort,
+                                           :resource_class => Katello::ProductContent))
+    end
+
+    def default_sort(relation)
```

Comment:
I would ordinarily put methods like default_sort and index_relation into the private section at the bottom of the controller, and it seems like that pattern is established in this controller.